mitodl/mit-open
# Make space for new homepage / pages

#### What are the relevant tickets?
Toward #27 

#### What's this PR do?
This PR:
- moves all the old `/infinite` pages to a separate /infinite-pages frolder. 
- Takes the /infinite baseurl off of `renderTestApp` and `renderWithProviders` testing function utilities
- reorganizings App.tsx a little
    - This caused a typescript error that I did not understand. Updating to 5.1.6 fixed the error and everything else seems to be working fine.

There's not much here besides moving files around, but I don't want the subsequent PR to show a bazillion file changes.

#### How should this be manually tested?
Check that...
1. http://localhost:8063/infinite still shows the infinite homepage
2. http://localhost:8063/infinite/cats shows 404
3. http://localhost:8063/cats shows 404 page and also 404 status code *You won't see the react app unless DEBUG is false.*
4. http://localhost:8063/ shows a very minimal homepage. (It just says "Home")
